src/pages/ProteinPage.tsx

```diff
@@ -0,0 +1,175 @@
+    index: number;
+    value: number;
+  }
+
+const valueToTabs = [
+    "details",
+    "feature_viewer",
+    "publications"
+]
+
+const tabsToValue:Record<string, number> = {
+    "details": 0,
+    "feature_viewer": 1,
+    "publications": 2,
+}
+  
+function TabPanel(props: TabPanelProps) {
+    const { children, value, index, ...other } = props;
+  
+    return (
+      <div
+        role="tabpanel"
+        hidden={value !== index}
+        id={`simple-tabpanel-${index}`}
+        aria-labelledby={`simple-tab-${index}`}
+        {...other}
+      >
+        {value === index && (
+          <Box sx={{ p: 3 }}>
+            <Typography>{children}</Typography>
+          </Box>
+        )}
+      </div>
+    );
+}
+  
+function a11yProps(index: number) {
+    return {
+      id: `simple-tab-${index}`,
+      'aria-controls': `simple-tabpanel-${index}`,
+    };
+}
+
+type CustomElement<T> = Partial<T & DOMAttributes<T> & { children: any }>;
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      ['protvista-uniprot']: CustomElement<ProtvistaUniprot>;
+    }
+  }
+}
+
+export const ProteinPage = () => {
+    const { id } = useParams();
+    const [proteinInfo, setProteinInfo] = useState<ProteinInfo>();
+    const [searchParams, setSearchParams] = useSearchParams();
+
+    const value = tabsToValue[searchParams.get("tab") ?? "details"]
+
+    useEffect(() => {
+        if (searchParams.get("tab")) {
+
+        } else {
+            setSearchParams(searchParams => {
+                searchParams.set("tab", "details")
+                return searchParams
+            })
+        }
+    }, [])
+    
+
+    const handleChange = (_: React.SyntheticEvent, newValue: number) => {
+        setSearchParams(searchParams => {
+            searchParams.set("tab", valueToTabs[newValue])
+            return searchParams
+        })
+    };
+
+    useEffect(() => {
+        async function fetch() {
+            const respProtein: ProteinInfo = await createPolymerInfoObject(id!);
+            setProteinInfo(respProtein);
+        }
+        fetch();
+    }, [])
+
+    return (
+        <div className="pageContainer">
+            <div className="infoHeader">
+                <h2 className="proteinHeader">{proteinInfo?.entry} / {proteinInfo?.entryNames}</h2>
+                <div className="organismObject">{proteinInfo?.organism}</div>
+            </div>
+            <div className="shortInfo">
+                <h5 className="shortInfoHeader">Protein</h5>
+                <p className="shortInfoContent">{proteinInfo?.proteinDescription}</p>
+                <h5 className="shortInfoHeader">Gene</h5>
+                <p className="shortInfoContent">{proteinInfo?.gene}</p>
+            </div>
+            <Box sx={{ borderBottom: 1, borderColor: 'divider', marginTop: '12px' }}>
+                <Tabs value={value} onChange={handleChange} aria-label="basic tabs example" >
+                    <Tab label="Details" {...a11yProps(0)} />
```

Comment:
```suggestion
                    <Tab value='details' label="Details" {...a11yProps(0)} />
```
and you don't need tab index at al.